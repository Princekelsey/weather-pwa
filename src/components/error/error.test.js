import React from "react";
import Error from "./Error";
import renderer from "react-test-renderer";

it("matches the snapshot", () => {
  const tree = renderer.create(<Error />).toJSON();
  expect(tree).toMatchSnapshot();
});
