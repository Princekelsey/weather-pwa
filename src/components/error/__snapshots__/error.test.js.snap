// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="center text-white"
>
  <h3>
    Error getting Weatherinformation
  </h3>
  <p>
    Please make sure your location gps is enabled and give application access
  </p>
</div>
`;
